src/Form/AdminSettingsForm.php

```diff
@@ -387,6 +387,21 @@ private function buildAdvancedConfigurationForm() {
       '#options' => ['trusted' => t('Trusted'), 'untrusted' => t('Untrusted')],
     ];
 
+    $form['inspector_status'] = [
+      '#type' => 'checkbox',
+      '#title' => t('Enable Lift Inspector'),
+      '#default_value' => $settings['inspector_status'],
+      '#description' => t('Enabling this feature will allow you to use type "cntl + i" to get lift information about the session'),
+    ];
+    $form['profile_manager_url'] = [
```

Comment:
I agree, I would have much preferred to get that information directly from the JS but it is not available and this was the only method available to me. I didn't want that to stop me from making progress.